docs/insights-client.8

```diff
@@ -40,6 +40,16 @@ Print archive to stdout; sets --silent and --no-upload.
 Specifies the compression algorithm to use. Choices are gz, bz2, xz, and none. Defaults to gz.
 .IP "--offline"
 Collect locally only, do not connect to Insights and do not upload.
+.IP "--analyze-container"
```

Comment:
I would just note that in the core client code (config.py) that --analyze-container, --analyze-image-id, --analyze-file and --analyze-mountpoint have been suppressed. So they will show up in the man pages but not in --help.